Uplift/Views/Home/FitnessCenterView.swift

```diff
@@ -16,6 +16,17 @@ struct FitnessCenterView: View {
     let fc: Facility?
 
     @StateObject private var viewModel = ViewModel()
+    @State private var fetchedEqmt: [Equipment] = Equipment.dummyData
+    /// Dictionary to track the number of equipment objects in each EquipmentType category
```

Comment:
`///` is reserved for official documentation. We're removing all of this anyways, but for next time, you want to use `//` instead.